back/app/serializers/web_api/v1/phase_serializer.rb

```diff
@@ -15,6 +15,8 @@ class WebApi::V1::PhaseSerializer < WebApi::V1::ParticipationContextSerializer
     user_basket object, params
   end
 
+  has_one :report, serializer: ReportBuilder::WebApi::V1::ReportSerializer
```

Comment:
@luucvanderzee The report data will be included in the phase serializer https://developers.citizenlab.co/api-docs/ee/frontweb_api/TAN-447-save-phase-reports/phases/get_one_phase_by_id.html

Reply:
I can see it in the relationships, is that what you mean? I don't see it in the `included` array

Reply:
> I can see it in the relationships, is that what you mean?

Yes, exactly.

> I don't see it in the included array

`included` array is used to avoid fetching this "included" data with a second HTTP call. Because "included" data is included in the first HTTP call.

But for now, I believe we don't need it as reports will be rarely fetched. Or at least only on pages for only one phase. In this case, it's fine to make a second HTTP call.